cmake/functions.cmake

```diff
@@ -139,5 +139,11 @@ endmacro()
 
 macro(google_enable_testing)
   enable_testing()
+  # For Ubuntu 20.04, finding gmock always throws a warning about
```

Comment:
Are these changes still needed? On CI it works without them when installing the `libgmock-dev` package.

Reply:
Without this particular change it does work, it just always throws a warning.  I was trying to quiet that down, but I can remove it if you don't think we should have it.

Reply:
Yes, I think this needs be removed.

Reply:
All right, removed in 864edf6.